src/rand.rs

```diff
@@ -208,23 +208,19 @@ mod sysrand {
 ))]
 mod urandom {
     use crate::error;
+    use spin::Once;
     use std;
 
     pub fn fill(dest: &mut [u8]) -> Result<(), error::Unspecified> {
-        use lazy_static::lazy_static;
+        static OPEN: Once<Result<std::fs::File, std::io::Error>> = Once::new();
```

Comment:
Here, wouldn't it make more sense to use `std::sync::Once` instead of `spin::Once`? We really don't want threads spinning while we wait for the file open operation to complete (and we can't try to open the file more than once concurrently, because we want to minimize the number of file handles we try to acquire).